packages/cli/src/loaders/index.spec.ts

```diff
@@ -860,6 +860,100 @@ user.password=Contraseña
     });
   });
 
+  describe('vtt bucket loader', () => {
+    it('should load complex vtt data', async () => {
+      setupFileMocks();
+  
+      const input = `
+  WEBVTT
+
+00:00:00.000 --> 00:00:01.000
+Hello world!
+
+00:00:30.000 --> 00:00:31.000 align:start line:0%
+This is a subtitle
+
+00:01:00.000 --> 00:01:01.000
+Foo
+
+00:01:50.000 --> 00:01:51.000
+Bar
+      `.trim();
+      
+      const expectedOutput ={
+        "0#0-1##": "Hello world!",
+        "1#30-31##align:start line:0%": "This is a subtitle",
```

Comment:
let's not make `align:start line ` part of the key, feels like it's irrelevant as it's just metadata